assembl/static2/js/app/components/search/UserMessagesTagFilter.jsx

```diff
@@ -0,0 +1,32 @@
+import { TagFilter } from 'searchkit';
+
+export default class UserMessagesTagFilter extends TagFilter {
+  handleClick = () => {
+    const { value } = this.props;
+    const typeAccessor = this.searchkit.accessors.statefulAccessors.type;
+    const qAccessor = this.searchkit.accessors.queryAccessor;
+    const creatorIdAccessor = this.searchkit.accessors.statefulAccessors.creator_id;
+    if (!qAccessor) {
+      console.error('Missing accessor for q in UserMessagesTagFilter'); // eslint-disable-line no-console
```

Comment:
Acceptable for now. Big TODO as DEVAS-1643 is coming to an end: Add support for Raven in React.

Reply:
yes, this component is mostly a copy/paste of `TagFilter` from searchkit except the logic of managing 3 different filters